lib/ui.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module UserInterface
+  def display_board(the_board)
+    puts " #{the_board[0]} | #{the_board[1]} | #{the_board[2]} "
```

Comment:
#### `REQUIRED`

Let's move all user interaction (`puts`, `print`, `gets.chomp`, `p`, etc) to `main.rb`.

> Add **executable** file named `main.rb` to `bin` directory. **This file should be responsible for user interaction. This is the only one place where `puts` and `get.chomp` (`print` and `p`) can be used.**


HINT: Let's move this to `main.rb` for this milestone. You can always return the `string` and print it in `main.rb`